mlrun/datastore/targets.py

```diff
@@ -1026,7 +1028,7 @@ def get_table_object(self):
         from storey import Table, V3ioDriver
 
         # TODO use options/cred
-        endpoint, uri = parse_v3io_path(self.get_target_path())
+        endpoint, uri = parse_path(self.get_target_path())
```

Comment:
A general comment on the various locations where `parse_v3io_path` was replaced with `parse_path` - up until now, any path not beginning with `v3io://` would return empty endpoint. Now if it's redis, it return something like `redis://<netloc>`. I think it is the safe thing to do to explode if such an endpoint is returned in code that works specifically with v3io - for example here, which passes the `endpoint` to `V3ioDriver` - the code there does check to see if there's an endpoint, but it doesn't check for cases where it begins with `redis://`.
One way would be to pass a parameter to `parse_path` which tells it whether a `redis` path is valid - if this param is `False` the function can return `None` like it did before, and this will maintain the same functionality.

Reply:
are you sure? I think it was raising an InvalidArgument exception..
btw, at least in this point I have the code separated by driver (because of the base class introduced)